styles/styles.css

```diff
@@ -782,3 +782,125 @@ main .section.small-vertical-margins {
         text-align: left;
     }
 }
+
+.getting-started main .default-content-wrapper ul {
+    margin: 0;
+    padding: 10px;
+}
+
+.getting-started main .default-content-wrapper li {
+    padding-bottom: 10px;
+}
+
+.getting-started main .default-content-wrapper p {
+    margin: 0 0 10px;
+}
+
+.getting-started main .default-content-wrapper img {
+    transform: scale(0.6);
+}
+
+main .section[data-background] picture.section-background-img {
+    position: absolute;
+    top: 0;
+    left: 0;
+    bottom: 0;
+    right: 0;
+    object-fit: cover;
+    z-index: -2;
+    overflow: hidden;
+}
+
+main .section[data-background] picture.section-background-img img {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+}
+
+.getting-started main .default-content-wrapper p > strong:first-child:last-child {
+    font-size: 1.7rem;
+    color: var(--link-color);
+}
+
+main .section.centered p:has(picture:first-child:last-child) {
+    text-align: center;
+}
+
+.getting-started main .section.grey {
+    background-color: var(--background-grey-4);
+}
+
+.getting-started main .default-content-wrapper .button-container {
+    max-width: 180px;
+    margin: 25px auto 30px;
+    padding-bottom: 25px;
+    border-bottom: 1px solid var(--color-grey-5);
+}
+
+.getting-started main .default-content-wrapper .button-container a {
+    margin: auto;
+    font-size: 1.3rem;
+}
+
+@media (min-width: 900px) {
+    .getting-started main .default-content-wrapper {
+        max-width: 830px;
+        margin: auto;
+    }
+
+    .getting-started main .default-content-wrapper h2 {
+        max-width: 650px;
+        margin: 50px auto;
+        font-size: 3.2rem;
+    }
+
+    .getting-started main .default-content-wrapper img {
+        transform: scale(1);
+    }
+
+    .getting-started main .section.inhaler {
+        position: relative;
+        padding-bottom: 120px;
+    }
+
+    .getting-started .section.section.inhaler::before,
+    .getting-started .section.section.inhaler::after {
+        display: block;
+        content: " ";
+        width: 200px;
+        height: 1px;
+        border-top: 1px solid var(--color-grey-5);
+        position: absolute;
+        left: calc(50% - 100px);
+    }
+
+    .getting-started .section.section.inhaler::before {
+        transform: translateY(-40px);
+    }
+
+    .getting-started .section.section.inhaler::after {
+        transform: translateY(130px);
+    }
+
+    .getting-started main .section.inhaler .default-content-wrapper {
+        max-width: 460px;
+        margin: auto;
+    }
+
+    .getting-started main .section.inhaler img:first-of-type {
+        position: absolute;
+        right: 311px;
```

Comment:
the image is displayed over the text
![image](https://github.com/hlxsites/bevespi/assets/26941572/5f4d486b-85b6-40fa-8b7f-685979e8b0b0)